### ISSUE TITLE
590 Database error (Note: ForestDB error code = -39)

### ISSUE THREAD

**Issue by sethrosetter** [couchbase/couchbase-lite-java-core#1297]
This test failed with 0.0.0-756 -  http://uberjenkins.sc.couchbase.com:8080/job/android-listener-client-sync-gateway/16/robot/report/log.html#s1-s2-t1

```
E/Router  (20817): Error updating doc: ls_db_9
E/Router  (20817): com.couchbase.lite.CouchbaseLiteException, Status: 590 (HTTP 500 Database error!)
E/Router  (20817): 	at com.couchbase.lite.store.ForestDBStore.add(ForestDBStore.java:831)
E/Router  (20817): 	at com.couchbase.lite.Database.put(Database.java:1837)
E/Router  (20817): 	at com.couchbase.lite.Database.putRevision(Database.java:1787)
E/Router  (20817): 	at com.couchbase.lite.Database.putRevision(Database.java:1778)
E/Router  (20817): 	at com.couchbase.lite.router.Router.update(Router.java:2138)
E/Router  (20817): 	at com.couchbase.lite.router.Router.update(Router.java:2191)
E/Router  (20817): 	at com.couchbase.lite.router.Router.do_PUT_Document(Router.java:2227)
E/Router  (20817): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/Router  (20817): 	at java.lang.reflect.Method.invoke(Method.java:511)
E/Router  (20817): 	at com.couchbase.lite.router.Router.start(Router.java:568)
E/Router  (20817): 	at com.couchbase.lite.listener.LiteServlet.service(LiteServlet.java:141)
E/Router  (20817): 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:802)
E/Router  (20817): 	at Acme.Serve.Serve$ServeConnection.runServlet(Serve.java:2251)
E/Router  (20817): 	at Acme.Serve.Serve$ServeConnection.parseRequest(Serve.java:2176)
E/Router  (20817): 	at Acme.Serve.Serve$ServeConnection.run(Serve.java:1988)
E/Router  (20817): 	at Acme.Utils$ThreadPool$PooledThread.run(Utils.java:1223)
E/Router  (20817): 	at java.lang.Thread.run(Thread.java:856)
```

Full logcat - 
https://gist.github.com/sethrosetter/4ad20935d537c94fefe137175449f07d#file-gistfile1-txt-L6179-L6197

**Comment by hideki** [comment]
I believe duplicate with https://github.com/couchbase/couchbase-lite-android/issues/909 and #1270. We should fix this for 1.3 release

**Comment by hideki** [comment]
This is duplicate issue with  couchbase/couchbase-lite-android#909. I will track this issue with 909. I will close this.
